src/redux/userSlice.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const userSlice = createSlice({
+  name: 'user',
+  initialState: {},
+  reducers: {
+    setUser: (state, action) => {
+      return action.payload;
+    },
+    clearUser: () => {}
+  }
+});
+
+export const { setUser, clearUser } = userSlice.actions;
+export const selectUser = state => state.user;
+export default userSlice.reducer;
```

Comment:
create a folder called user.
Inside the folder, there should be 3 files: userReducer, userSelectors, and userActions.
Split what's in this file into those respective files.
This reducer should be called userReducer in the name and the const variable